controllers/utils.js

```diff
@@ -9,10 +9,18 @@ const AppConstants = require("../app-constants");
 
 
 function generatePageToken(req) {
-  const pageToken = {ip: req.ip, date: new Date(), nonce: uuidv4()};
-  const cipher = crypto.createCipher("aes-256-cbc", AppConstants.COOKIE_SECRET);
-  const encryptedPageToken = [cipher.update(JSON.stringify(pageToken), "utf8", "base64"), cipher.final("base64")].join("");
-  return encryptedPageToken;
+  const pageToken = { ip: req.ip, date: new Date(), nonce: uuidv4() };
```

Comment:
Not sure we need the `nonce` anymore - if I remember correctly, I was using it as make-shift 128-bit "salt" for each token. But a 16-byte IV should have the same effect?